gradio/utils.py

```diff
@@ -962,6 +962,16 @@ def abspath(path: str | Path) -> Path:
         return path.resolve()
 
 
+def is_in(path_1: str | Path, path_2: str | Path):
```

Comment:
I wonder if this should return true if the paths are equal - can help clean up some of the code

i.e

```
utils.is_in(abs_path, allowed_path) or abs_path == utils.abspath(allowed_path)
``